easybuild/easyconfigs/f/foss/foss-2019b.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'Toolchain'
+
+name = 'foss'
+version = '2019b'
+
+homepage = 'https://easybuild.readthedocs.io/en/master/Common-toolchains.html#foss-toolchain'
+description = """GNU Compiler Collection (GCC) based compiler toolchain, including
+ OpenMPI for MPI support, OpenBLAS (BLAS and LAPACK support), FFTW and ScaLAPACK."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+gccver = '8.3.0-2.32'
```

Comment:
@vanzod You proposed to rename this to just `8.3.0`, since we stick to a single `binutils` version per GCC(core) version anyway... And it leaves the door open for making exceptions, by adding back the suffix in case another `binutils` version has to be used for a particular reason.

Are you up for looking into making that change here?